src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -764,6 +765,10 @@ def clone_subvolume_snapshot(self, **kwargs):
         s_subvolname = kwargs['sub_name']
         s_groupname  = kwargs['group_name']
 
+        if get_all_pending_clones_count(self, self.mgr, self.volspec) >= self.mgr.max_concurrent_clones:
```

Comment:
This is outside the try block and if get_all_pending_clones_count returns `ve.errno, None` how would this behave?

Reply:
I am now raising exception in `get_all_pending_clones_count` and calling in try-catch block.